ES/ESmy020.xml

```diff
@@ -31,71 +31,394 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                      እድ፡ ላዕለ፡ ለሕጻን፡ ላዕለ፡ እሙ፡ ለሕፃን፡ ወእምድኅረ፡ ነጽሐት፡ በ፵፡ ዕለት፡ ትበውእ፡ ቤተ፡ ክርስቲያን፨ ወያነብቡ፡ ካህናት፡
+                      ጸሎተ፡ አኰቴት፡ ወጸሎተ፡ ዕጣን፨ ወእምድኅሬሁ፡ ዘንተ፡ ጸሎተ፨ እግዚአብሔር፡ አምላክነ፡ አኃዜ፡ ኵሉ፡ አቡሁ፡ ለእግዚእነ፡ ወመድኃኒነ፡
+                      ኢየሱስ፡ ክርስቶስ፡ ፈጣሬ፡ ዓላማት፡ ዘገበርከ፡ ልያተነ፨
+                        </q>
+                      </incipit>
+                       </msItem>
+            </msContents>
+
                <physDesc>
                   <objectDesc form="Codex">
                      <supportDesc>
                         <support>
                            <material key="parchment"/>
                         </support>
-                        <extent>Made of <measure unit="leaf">71.0</measure> folios, in <measure unit="quire">7.0</measure> quires . <dimensions type="outer">
+                        <extent>
+                          <measure unit="leaf">71</measure>
+                          <measure unit="quire">7</measure>
+                           <dimensions type="outer">
                               <height unit="mm">185</height>
                               <width unit="mm">160</width>
                               <depth unit="mm">40</depth>
                            </dimensions>
                         </extent>
                         <foliation><!--2016-10-18+02:00 Updated MY-020 from DomLib--></foliation>
                         <collation>
+                          <note></note>
                            <list>
-                              <item xml:id="q1">
-                                 <locus from="1r" to="6v"/>I(fols. 1r-6v) </item>
-                              <item xml:id="q2">
-                                 <locus from="7r" to="14v"/> II(fols. 7r-14v) </item>
-                              <item xml:id="q3">
-                                 <locus from="15r" to="22v"/> III(fols. 15r-22v) </item>
-                              <item xml:id="q4">
-                                 <locus from="23r" to="33v"/> IV(fols. 23r-33v) </item>
-                              <item xml:id="q5">
-                                 <locus from="34r" to="43v"/> V(fols. 34r-43v) </item>
-                              <item xml:id="q6">
-                                 <locus from="44r" to="53v"/> VI(fols. 44r-53v) </item>
-                              <item xml:id="q7">
-                                 <locus from="54r" to="71v"/> VII(fols. 54r-71v)&#xD; &#xD; Quire I
-                                 is composed entirely of fols. from other parts of the Ms., its
-                                 folios are attached as guard leaves by means of crude stitches. All
-                                 the folios of quire I belong to Text I.</item>
+                              <item xml:id="q1" n="1">
+                                 <locus from="1r" to="5v"/>
+                                 <dim unit="leaf">4</dim>
+                                  <locus from="1r" to="14v"/>
```

Comment:
where does this second locus come from

Reply:
you can only describe the composition in sequence as it is now

Reply:
no idea, removed